### PR TITLE
[DOCS] Add Zcb Instructions in CVA6 user guide and requirement specification

### LINKED ISSUE #1411: [TASK] Add Zc* in user guide and Requirement Specification
### Is there an existing CVA6 task for this?

- [x] I have searched the existing task issues

### Task Description

When Zc RTL code is merged, update the requirement specification and user guide (instruction description included) to mention Zc* and Zicond extension support

### Required Changes

Update Requirement spec & user manual

### Current Status

Zc is not mentioned in the specs

### Risks

_No response_

### Prerequisites

_No response_

### KPI (KEY Performance Indicators)

_No response_

### Description of Done

merged PR

### Associated PRs

_No response_

### PR DESCRIPTION
**Pull request by Abdulwadoodd**
This PR (partially) resolves #1411

### CODE REVIEW

**Comment by JeanRochCoulon** [comment]
@AyoubJalali  Can you approve it ?

**Comment by AyoubJalali** [comment]
@Abdulwadoodd great work !!
Just to keep the same format of the spec, could you remove     
    
    **Group**: Zca

    **Prerequisites**: NONE

**Comment by AyoubJalali** [comment]
@Abdulwadoodd also can you separate the ZC extension section from compressed section, and you could mention that it required the C extension

File: docs/01_cva6_user/RISCV_Instructions.rst [hunk 1]
```diff
@@ -1126,6 +1193,122 @@ Integer Computational Instructions
 
     **Exception raised**: Raise a Breakpoint exception.
 
+    **Group**: Zca
+
+    **Prerequisites**: NONE
+
+- **C.ZEXT.B**: Compressed Zero Extend Byte
+
+    **Format**: c.zext.b rd''
```
Comment (AyoubJalali) [inline docs/01_cva6_user/RISCV_Instructions.rst:1202]: it's just rd', not rd''

File: docs/01_cva6_user/RISCV_Instructions.rst [hunk 2]
```diff
@@ -1126,6 +1193,122 @@ Integer Computational Instructions
...
+
+    **Exception raised**: NONE
+
+    **Group**: Zcb
+
+    **Prerequisites**: NONE
+
+- **C.MUL**: Compressed Multiply
+
+    **Format**: c.mul rsd', rs2'
```
Comment (AyoubJalali) [inline docs/01_cva6_user/RISCV_Instructions.rst:1298]: what do you mean by **_rsd'_** ?

File: docs/01_cva6_user/RISCV_Instructions.rst [hunk 3]
```diff
@@ -1126,6 +1193,122 @@ Integer Computational Instructions
...
+
+    **Description**: performs a 32-bit × 32-bit multiplication and places the lower 32 bits in the destination register (Both rs1 and rs2 treated as signed numbers).
```
Comment (AyoubJalali) [inline docs/01_cva6_user/RISCV_Instructions.rst:1300]: it's rd and rs2 treated as signed numbers

**Comment by Abdulwadoodd** [comment]
Hi @AyoubJalali ,
Docs have been updated accounting for the above comments.

**Review by AyoubJalali — APPROVED** [review]

**Comment by AyoubJalali** [comment]
> Hi @AyoubJalali , Docs have been updated accounting for the above comments.

thanks

**Pull request merged**